src/main/java/seedu/address/model/entity/fridge/Status.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.entity.fridge;
+
+//@@author arjavibahety
+/**
+ * Enumerates all the possible states a fridge undergoes in the mortuary.
+ */
+public enum Status {
```

Comment:
Is it better if this is renamed to `FridgeStatus`? Because `Body` class already has a `Status` enum

Reply:
Then should status for the body be called `BodyStatus` ?

Reply:
I'll change it if needed!